diagnostic.md

```diff
@@ -45,18 +54,22 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
+The 1st 'weather' expression sets 'weather' equal to the value associated with "sunny"
+The 2nd 'weather' experssion sets the absolute value of 'weather' to 'sunny' as a string
+
+
 
 ## Question 5
 
 ```js
 //We'll learn about require later in the course
 const ask = require('../lib/ask.js');
 
-let answer = 'not empty';
+let answer = '1';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer === '1') {
    console.log("You guessed it!");
    break;
   }
```

Comment:
also, you were supposed to put this in bin/diagnostic.js so that you can run the code